tests/file.go

```diff
@@ -67,11 +68,13 @@ func RunFileTest(rootDir string, path string, nativeLibs bool, logger loggerFunc
 	stdin := new(bytes.Buffer)
 	stdout := new(bytes.Buffer)
 	stderr := new(bytes.Buffer)
-	filesPath := "./files"
+	filesPath := "./files2"
+	mode := ImportModeStdlibsPreferred
 	if nativeLibs {
```

Comment:
this was weird. looks like before there was a bug where the native functions were only being run in files2 (when it should be so for files1 instead), and somehow the tests were still passing anyways? well, either way, the intent was this.